devtools/x/src/utils.rs

```diff
@@ -32,6 +32,7 @@ pub fn locate_project(cargo_config: &CargoConfig) -> Result<PathBuf> {
     Ok(serde_json::from_slice::<LocateProject>(&output)?.root)
 }
 
+#[allow(dead_code)]
```

Comment:
any reason to not remove?

Reply:
Yeah I was thinking it might be useful in the future, but probably not given this approach. Will remove.

Reply:
Removed.